package.json

```diff
@@ -21,12 +21,13 @@
     "watch": "node tasks/watch.js",
     "test": "mocha src/**/*spec.js",
     "test:ci": "nyc --reporter=lcov mocha",
-    "stylelint": "stylelint src/scss/**/*.scss src/scss/*.scss --color --formatter verbose",
-    "eslint": "eslint src/js",
-    "phplint": "vendor/squizlabs/php_codesniffer/bin/phpcs --standard=sparkpress-standard.xml --extensions=php ./src/php",
-    "node-version": "check-node-version --package",
-    "twiglint": "twigcs src/php/views",
-    "lint": "run-p phplint stylelint eslint twiglint"
+    "lint:js": "eslint src/js",
+    "lint:php": "./scripts/run.sh vendor/bin/phpcs --standard=sparkpress-standard.xml --extensions=php src/php",
+    "lint:scss": "stylelint src/scss/**/*.scss src/scss/*.scss --color --formatter verbose",
+    "lint:twig": "./scripts/run.sh vendor/bin/twigcs src/php/views",
+    "lint": "run-p lint:*",
+    "run:php": "./scripts/run.sh",
```

Comment:
Minor quibble: I got tripped up when typing `npm run run:php`. It just feels weird (to me) to type the same verb twice in a row. Maybe `npm run php:run` would be better? No need to change it if you don't want, it could just be how my brain works 🙃

Reply:
It felt weird to me too. Updated!